tsconfig.json

```diff
@@ -1,6 +1,8 @@
 {
   "extends": "./node_modules/@jupiterone/integration-sdk-dev-tools/config/typescript",
   "compilerOptions": {
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
```

Comment:
Isn't this redundant? I thought setting `esModuleInterop`, also sets `allowSyntheticDefualtImports` by default.

Reply:
Removed in latest commit.  Thanks!